docs/source/gcp/examples/iam-role-create.rst

```diff
@@ -0,0 +1,33 @@
+IAM - Watch for Creation of New Roles
+============================================================
```

Comment:
having 200 examples of the same thing isn't particular useful. let's stop with the resource create, notify examples we have enough.

Reply:
okay, I will close this PR and after actions, implementation will provide a better example

Reply:
okay, I will close this PR and after actions, implementation will provide a better example